src/CMakeLists.txt

```diff
@@ -1,9 +1,3 @@
-# Collect source files into the "sources" variable and unit test files into the
-# "gtest_sources" variable
 ign_get_libsources_and_unittests(sources gtest_sources)
 
-# Create the library target
 ign_create_core_library(SOURCES ${sources} CXX_STANDARD ${c++standard})
```

Comment:
can we get rid of `SOURCES` if we declare it as `INTERFACE`?

Reply:
humm I see that's what `ign-cmake` uses, we could give it a try

Reply:
I would be fine with it either way